anime-list-master.xml

```diff
@@ -30557,7 +30557,7 @@
   <anime anidbid="11135" tvdbid="289763" defaulttvdbseason="1" episodeoffset="13" tmdbid="" imdbid="">
     <name>The Idolmaster: Cinderella Girls (2015)</name>
     <mapping-list>
-      <mapping anidbseason="0" tvdbseason="0" start="2" end="4" offset="1"/>
+      <mapping anidbseason="0" tvdbseason="0" start="2" end="3" offset="0">;1:4;</mapping>
```

Comment:
Wrong separator, should be a `-` instead of `:`.
```suggestion
      <mapping anidbseason="0" tvdbseason="0" start="2" end="3" offset="0">;1-4;</mapping>
```